### PR TITLE
Add about mode document for website

### LINKED ISSUE #12065: Adjust governance doc to cluster mode
- [ ]  Rewrite governance doc to cluster mode
- [ ] Explain new concept - mode
- [ ] Design doc structure via new mode(memory, standalone and cluster)

### PR DESCRIPTION
**Pull request by zjcnb**
Fixes #12065 

image
image

### CODE REVIEW

File: docs/document/content/features/mode/_index.cn.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+chapter = true
++++
+
+## 背景
+
+为满足用户不同的需求，比如快速测试、单机运行和分布式运行。mode 提供了三种模式，它们分别是 Memory 模式、Standalone 模式、以及 Cluster 模式。
+
+## Memory 模式
+
+Memory 模式比较适用于做快速集成测试，方便测试、运维及开发人员做快速的整合功能测试。该模式也是 Apache ShardingSphere 的默认模式。
```
Comment (terrymanu) [inline docs/document/content/features/mode/_index.cn.md:14]: Memory 模式适用于做快速集成测试，方便开发人员在整合功能测试中集成 ShardingSphere。该模式也是 Apache ShardingSphere 的默认模式。

File: docs/document/content/features/mode/_index.cn.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Standalone 模式
+
+Standalone 模式比较适用于单机环境下使用，通过该模式可将数据源、规则、元数据进行持久化。其中 Standalone 模式中的 File 属性会将配置信息写入您所指定的 Path，
```
Comment (terrymanu) [inline docs/document/content/features/mode/_index.cn.md:18]: Standalone 模式适用于单机环境，通过该模式可将数据源、规则等元数据进行持久化

Comment (terrymanu) [reply docs/document/content/features/mode/_index.cn.md:18]: 属性 or 类型?

File: docs/document/content/features/mode/_index.cn.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+如果没有设置 Path 属性，那么 ShardingSphere 会在根目录创建 .shardingsphere 文件用来存储配置信息。
+
+## Cluster 模式
+
+Cluster 模式比较适用于分布式场景下使用，Cluster 模式提供了多个实例之间共享元数据，节点状态的同步，以及通过 Dist SQL 动态的调整规则等功能。
```
Comment (terrymanu) [inline docs/document/content/features/mode/_index.cn.md:23]: Cluster 模式适用于分布式场景，它提供了多个计算节点之间的元数据共享和状态协调。

File: docs/document/content/features/mode/_index.cn.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+关于分布式治理相关可[点击这里](https://shardingsphere.apache.org/document/current/cn/features/governance/)
```
Comment (terrymanu) [inline docs/document/content/features/mode/_index.cn.md:24]: This should be removed and move it to cluster mode

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: docs/document/content/features/mode/_index.cn.md (hunk 4 above)
Comment (zjcnb) [reply docs/document/content/features/mode/_index.cn.md]: done

File: docs/document/content/features/mode/_index.cn.md (hunk 2 above)
Comment (zjcnb) [reply docs/document/content/features/mode/_index.cn.md]: change to 类型

File: docs/document/content/features/mode/_index.cn.md (hunk 1 above)
Comment (zjcnb) [reply docs/document/content/features/mode/_index.cn.md]: done

**Comment by mspaansen** [comment]
😀

**Comment by ThanoshanMV** [comment]
> Fixes #12065
> 
> image
> image

Hi @zjcnb, were the above images taken while testing in the local environment?

**Comment by zjcnb** [comment]
> > Fixes #12065
> > image
> > image
> 
> Hi @zjcnb, were the above images taken while testing in the local environment?

yes, you can execute build.sh in doc, then it will be generated file in the target directory

**Comment by ThanoshanMV** [comment]
Thank you @zjcnb

**Pull request merged**